buildbot/dependency.conf

```diff
@@ -1,8 +1,8 @@
 [VERSIONS]
 # https://github.com/intel/llvm/releases/download/2020-03/oclcpuexp-2020.10.4.0.15_rel.tar.gz
```

Comment:
The comment should contain link to binaries archive.
Please work with @tfzhu to prepare dependency release with Windows and Linux CPU RTs.